packages/formdescriptioneditors/frontend/sirius-components-formdescriptioneditors/src/PageList.tsx

```diff
@@ -395,35 +396,30 @@ export const PageList = ({ editingContextId, representationId, formDescriptionEd
                   </div>
                 }
                 key={page.id}
-                onClick={handleClick}
-                onKeyDown={handleDelete}
-                draggable={true}
-                onDragStart={handleDragStart}
-                onDragEnter={handleDragEnter}
-                onDragOver={handleDragOver}
-                onDragLeave={handleDragLeave}
-                onDrop={handleDropTab}
+                onClick={readOnly ? noop : handleClick}
+                onKeyDown={readOnly ? noop : handleDelete}
+                draggable={!readOnly}
+                onDragStart={readOnly ? noop : handleDragStart}
```

Comment:
May be it would have been better/more readable to modify handleDragEnter/handleDragOver/handleDragLeave... instead.
What do you think about it?